readme.md

```diff
@@ -28,7 +54,7 @@ VITE_PROJECT_ID="your_project_id"
 3. Run `pnpm watch` to build and watch for file changes in a separate tab
 4. Run gallery, laboratory or examples in a separate tab i.e. `pnpm laboratory`
 
-# Releasing new versions
+## Releasing new versions
```

Comment:
Let's not give some info about any part of the development of this repo, this readme should be only for advertisement. Development and other details should be in another doc